data/helpers.d/filesystem

```diff
@@ -80,3 +83,23 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Restore a file or a directory to the stored path
+#
+# Use the registered path in backup_list by ynh_backup to restore the file at the good place.
+#
+# usage: ynh_restore_file file_to_restore
+# | arg: file_to_restore - File or directory to restore
+ynh_restore_file () {
```

Comment:
I am ok for adding this hook but I think we should use a "ynh_restore" that restore all backuped files directly .

ynh_restore_file should be reserved for particular cases.